docs/_Docs_GetStarted.md

```diff
@@ -35,7 +35,51 @@ Alternatively, you can start a local test node and deploy the contracts yourself
 ```
 ~$ yarn run start:blockchain:client
 
-~$ ./node_modules/.bin/truffle migrate
+~$ ./node_modules/.bin/truffle migrate --reset --compile-all
 ```
 
 For more detailed instructions, and additional steps required to set up an environment for use with colonyJS, refer to the [colonyJS get started doc](/colonyjs/docs-get-started/).
+
+## Set Up Reputation Mining for Local Testing
+
+The Reputation Mining client is usable for testing, but has a limited functionality. There is currently no challenge-response process to accommodate multiple submitted Reputation Root Hashes. Still, it is possible to run a single miner instance for usable reputation scores on a testnet.
+
+### Start the Mining Client
+
+Start the mining client with:
+
+```
+node packages/reputation-miner/bin/index.js --file ./reputations.json --colonyNetworkAddress 0x76d508fa65654654ffdb334a3023353587112e09 --minerAddress 0xb77d57f4959eafa0339424b83fcfaf9c15407461
+```
+
+This will use the first account in `ganache-accounts.json` as the Reputation miner.
```

Comment:
The `minerAddress` in the execution above is the first account in `ganache-accounts.json`